session_2/student_registry/db/migrate/20210114031001_add_fields_to_students.rb

```diff
@@ -1,7 +1,7 @@
 class AddFieldsToStudents < ActiveRecord::Migration[6.1]
   def change
     add_column :students, :branch, :string
-    add_column :students, :cgpa, :decimal, precision:2, scale: 2
+    add_column :students, :cgpa, :decimal, precision:4, scale: 2
```

Comment:
With a precision of 4, I can save values like `15.00`

Can you write validations to ensure that CGPA is between 0 and 10?

https://guides.rubyonrails.org/active_record_validations.html